plugins/woocommerce/includes/class-wc-install.php

```diff
@@ -1232,6 +1233,45 @@ private static function should_enable_hpos_for_new_shop() {
 		return apply_filters( 'woocommerce_enable_hpos_by_default_for_new_shops', true );
 	}
 
+	/**
+	 * Get the wc_order_stats table schema.
+	 *
+	 * @since 10.4.0
+	 * @param string $collate Database collation.
+	 * @return string SQL schema for wc_order_stats table.
+	 */
+	private static function get_order_stats_table_schema( $collate ) {
+		global $wpdb;
+
+		$is_new_install = self::is_new_install();
```

Comment:
We should also check here if the `fulfillment_status` column already exists or not, given this function would get run on any subsequent DB updates, so a new store may already have the new column, but is not new anymore when a new DB update happens.
This didn't cause any particular warnings for me, but should be addressed.